src/services/aws.js

```diff
@@ -0,0 +1,30 @@
+import AWS from 'aws-sdk/global'
+import 'aws-sdk/clients/dynamodb'
+
+const ROLE_ARN = 'arn:aws:iam::407461997746:role/vueaws'
+
+let AwsDocClient = null
+let awsDocClientResolve = null
+const awsDocClientInit = new Promise((resolve) => awsDocClientResolve = resolve)
```

Comment:
A little bit hacky, but was an easy solution for making sure the credentials have been set on first page load